src/cross_module_fn.h

```diff
@@ -98,6 +98,7 @@ typedef struct CrossModuleFunctions
 	PGFunction compressed_data_out;
 	bool (*process_compress_table)(AlterTableCmd *cmd, Hypertable *ht,
 								   WithClauseResult *with_clause_options);
+	void (*process_altertable_cmd)(const Hypertable *ht, const AlterTableCmd *cmd);
```

Comment:
agree with erik that ideally this should happen in tsl code to not further bloat the cross module stuff

Reply:
The alter table cmd processing has been moved to tsl code. This is the function from cross_module that would reference tsl code.